### PR TITLE
move shared dependencies to common

### PR DESCRIPTION
**Pull request by xvrl**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,10 +70,12 @@
         <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
         <maven-site-plugin.version>3.7.1</maven-site-plugin.version>
         <maven-project-info-reports-plugin.version>3.0.0</maven-project-info-reports-plugin.version>
+        <mockito.version>2.23.0</mockito.version>
         <mockito-all.version>1.10.19</mockito-all.version>
```
Comment (ijuma) [inline pom.xml]: Is anything still using Mockito 1.10?

File: pom.xml [hunk 2]
```diff
@@ -231,6 +234,11 @@
                 <artifactId>slf4j-log4j12</artifactId>
                 <version>${slf4j.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.code.gson</groupId>
+                <artifactId>gson</artifactId>
+                <version>${gson.version}</version>
+            </dependency>
```
Comment (ijuma) [inline pom.xml]: Why is this under `kafka deps`?

File: pom.xml (hunk 1 above)
Comment (xvrl) [reply pom.xml]: Yes we have some private repos that use it

File: pom.xml (hunk 2 above)
Comment (xvrl) [reply pom.xml]: Fixed

**Pull request merged**